app.py

```diff
@@ -1,9 +1,41 @@
+import uuid
 from flask import render_template
 
 from server import create_app, run_app, db, socketio
 from server.models import AuthType, Users, Projects, Sources
 
 
+
+
+def new_google_user(profile):
+    user_name = profile["name"]
+    email = profile["email"]
+    profile_picture = profile["imageUrl"]
+    auth_type = AuthType.GOOGLE
+    user_id = uuid.uuid4()
+    db.session.add(Users(user_id, user_name,auth_type,email,profile_picture));
```

Comment:
Be careful not to mix up syntax with JavaScript. Python does not need semicolons. :eyes: